.github/workflows/build.yml

```
@@ -98,3 +98,23 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ["1.18", "1.19"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
```

Consistent with the `lint` job.

```suggestion
      # We need the Go version and Go cache location for the actions/cache step,
      # so the Go installation must happen before that.
      - id: setup-go
```

---

@mcdonnnj resolved with [7751db5](https://github.com/cisagov/con-pca-tasks/pull/1/commits/7751db53ea6145e573bea708aea853b28cc8aea0)